core/src/main/scala/kanaloa/reactive/dispatcher/queue/Queue.scala

```diff
@@ -58,9 +59,7 @@ trait Queue extends Actor with ActorLogging with MessageScheduler {
       finish(status, s"Queue successfully retired")
       PartialFunction.empty //doesn't matter after finish, but is required by the api.
     } else handleWork(status, retiring) orElse {
-      case Enqueue(_, replyTo, _) ⇒
-        replyTo.getOrElse(sender) ! Retiring
-
+      case Enqueue         ⇒ sender() ! Retiring
```

Comment:
I've captured that in https://github.com/iheartradio/kanaloa/issues/75 along with https://github.com/iheartradio/kanaloa/issues/76 and will fix those in a different PR.